IssuePreviewCVCell.swift

```diff
@@ -11,26 +11,95 @@ import UIKit
 class IssuePreviewCVCell: UICollectionViewCell {
     static var reuseID = "IssuePreviewCVCell"
     
-    var issuePreviewView: IssuePreviewView = {
-        var issue = IssuePreviewView.nib()
-        issue.translatesAutoresizingMaskIntoConstraints = false
-        return issue
+    var containerView: UIStackView = {
+        let view = UIStackView()
+        view.axis = .vertical
+        view.alignment = .leading
+        view.distribution = .fill
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
     }()
     
-    func configure(issue: Issue){
-        issuePreviewView.configure(issue: issue)
-        addSubview(issuePreviewView)
-        // subView constraints added here
+    var containerHView: UIStackView = {
+        let view = UIStackView()
+        view.axis = .horizontal
+        view.alignment = .fill
+        view.distribution = .fill
+        view.spacing = 20
+        return view
+    }()
+    
+    var issueTitle: UILabel = {
```

Comment:
common nomenclature for UI components is including the type in the name, like `issueTitleLabel`.  `issueTitle` kind of sounds stringy, like this is the issue title, not the thing that is displaying the issue title.  (and this applies to things up and down here)